bude_hezky/main.py

```diff
@@ -0,0 +1,36 @@
+import argparse, os
+from dotenv import load_dotenv
+from bude_hezky.weather import weather_forecast, weather_resolver
+from bude_hezky.sender import email_sender
+
+CITY_KEY = 'mesto'
+EMAIL_KEY = 'email'
+CHAT_ID = 'chat'
+FROM_EMAIL = 'ivan@ikvasnica.com'
+
+parser = argparse.ArgumentParser()
+parser.add_argument(CITY_KEY, help='Město, kde bydlíš. Můžeš zkusit i vesnici.')
+parser.add_argument('--' + EMAIL_KEY, help='E-mail, na který pošleme informaci, zda bude hezky.')
+parser.add_argument('--' + CHAT_ID, help='Telegram ID konverzace, do které pošleme informaci o počasí.')
+cli_arguments = vars(parser.parse_args())
+
+load_dotenv()
+api_key = os.getenv('WEATHER_API_KEY')
+
+city = cli_arguments[CITY_KEY]
+to_email = cli_arguments[EMAIL_KEY]
+try:
+    tomorrow_forecast = weather_forecast.get_forecast_for_city(api_key, city)
+except weather_forecast.NoForecastRetrievedException as e:
+    print('Weather could not be retrieved due to an error: %s' % (e))
+    exit(1)
```

Comment:
Vůbec jsem nevěděl, že `exit()` funguje i jinde než v shellu. Zkoušel jsi, že to opravdu funguje? Myslel jsem, že [v programu je potřeba použít `sys.exit()`](https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python)

Reply:
Fungovalo, ale přepíšu :)